### ISSUE TITLE
Notification that Configure Failed, Debug with CMake Debugger shows up when you override a Configure with a Build

### ISSUE THREAD

**Issue by sinemakinci1** [microsoft/vscode-cmake-tools#3343]
### Brief Issue Summary

When a CMake Configure action is running, and you override with a Build action, I get an error that Configure Failed, Debug with CMake Debugger. Ideally, this would not surface to the user, since nothing "failed".

**Comment by cjdb** [comment]
There should also be a way to disable the pop-up for legitimate cases.

**Comment by gcampbell-msft** [comment]
@cjdb Thanks for the suggestion. Could you elaborate some on what you mean by "legitimate cases"? It'd be great to understand any and all cases where we should allow this.

**Comment by cjdb** [comment]
That would be whatever the reason the popup was designed for.

**Comment by hugsy** [comment]
> There should also be a way to disable the pop-up for legitimate cases.

Agreed 💯, that pop-up is extremely annoying. There should be an option to disable it.

**Comment by teo-tsirpanis** [comment]
The same error pops when I cancel configuring (#3472).

**Comment by cjdb** [comment]
The "Do not show again" button doesn't appear to have any effect. I still get the pop-up even after pressing that button (I've run CMake twice in a row after pressing the button to see if a change in my environment is the culprit, and it is not).

**Comment by gcampbell-msft** [comment]
@cjdb Also, what operating system are you on?

**Comment by cjdb** [comment]
> If you search for `cmake.showConfigureWithDebuggerNotification` in your settings, could you tell me what you see? I've just tested on the most recent pre-release whether this takes affect, and I've confirmed (for my repro) that it does take affect. I'm wondering if there is some settings issue that is happening?

That setting isn't present in my `settings.json`.

> @cjdb Also, what operating system are you on?

Arch Linux. I've also run the following, which may be relevant too?

```
$ code --version
1.86.1
31c37ee8f63491495ac49e43b8544550fbae4533
x64
```

**Comment by cjdb** [comment]
Update: I see `"cmake.showConfigureWithDebuggerNotification": false` on my work machine, which runs macOS. Perhaps my Arch VS Code build is out-of-date.

**Comment by gcampbell-msft** [comment]
@cjdb It might not be in your settings.json yet, but if you search for it in the global settings UI, you should be able to find it. 

Could you check there?

**Comment by cjdb** [comment]
Yep, it's there. Looks like the global settings UI is having difficulty writing to my `settings.json`, but I've been able to write to that manually, which seems to have solved my problem :)

My guess is that I've got a permissions issue or something, but that's out of scope for the bug!
